docs/components/tables/table.md

```diff
@@ -26,13 +28,20 @@ Users are provided with 2 different sized tables (SM and MD) in GEL or based on
 * The building blocks of table can be pieced together to create a table.
 * If the situation is sufficed with a table placehodler, consider using the two table components which would save time in return.
 * A pagination element in a table is displayed at the bottom of the table component when displayed.
+* Use [button control](../control-buttons/button-control.md) as table filters with the label floating when active or any value is selected.
+* The following behaviour takes place when a filter is being selected for a table.
+<Video source="../../../videos/filter-behaviour.mp4" />
```

Comment:
maybe include the path inside the Video component?

Reply:
the prop could be file name

Reply:
otherwise, it is a bit over-engineering. you can actually use html `<video>` instead of JSX

Reply:
yup understood. i thought there were a few things repeating that might become clumsy when additional videos were added, hence the jsx approach